helpdesk/email.py

```diff
@@ -554,22 +578,22 @@ def object_from_message(message, queue, logger):
             else:
                 name = ("part-%i_" % counter) + name
 
-            # FIXME: this code gets the paylods, then does something with it and then completely ignores it
-            # writing the part.get_payload(decode=True) instead; and then the payload variable is
-            # replaced by some dict later.
-            # the `payloadToWrite` has been also ignored so was commented
-            payload = part.get_payload()
-            if isinstance(payload, list):
-                payload = payload.pop().as_string()
-            # payloadToWrite = payload
-            # check version of python to ensure use of only the correct error type
-            non_b64_err = TypeError
-            try:
-                logger.debug("Try to base64 decode the attachment payload")
-                # payloadToWrite = base64.decodebytes(payload)
-            except non_b64_err:
-                logger.debug("Payload was not base64 encoded, using raw bytes")
-                # payloadToWrite = payload
+            # # FIXME: this code gets the paylods, then does something with it and then completely ignores it
+            # # writing the part.get_payload(decode=True) instead; and then the payload variable is
+            # # replaced by some dict later.
+            # # the `payloadToWrite` has been also ignored so was commented
+            # payload = part.get_payload()
+            # if isinstance(payload, list):
+            #     payload = payload.pop().as_string()
+            # # payloadToWrite = payload
+            # # check version of python to ensure use of only the correct error type
+            # non_b64_err = TypeError
+            # try:
+            #     logger.debug("Try to base64 decode the attachment payload")
+            #     # payloadToWrite = base64.decodebytes(payload)
+            # except non_b64_err:
+            #     logger.debug("Payload was not base64 encoded, using raw bytes")
+            #     # payloadToWrite = payload
```

Comment:
What is going on here? If the code is dead, then kill it. This is like gangrenous, rotting, zombie code at this point.

Reply:
I hope that one day someone who understands what happens gets it back; if removed then no chance to see it done